configuration.adoc

```diff
@@ -1701,7 +1701,14 @@ zmcontrol restart
 S/MIME is a standard to send secure email messages. S/MIME messages use
 digital signature to authenticate and encrypt messages.
 
-=== Prerequisites
+Currently, there are two versions of S/MIME feature
+
+. The old client based solution which requires Java 1.6 SE deployed on client machine
```

Comment:
I think it read better with "the" in front of "client machine".
Same comment for the next line.